db/migrate/20200727090506_create_posts.rb

```diff
@@ -0,0 +1,10 @@
+class CreatePosts < ActiveRecord::Migration[5.2]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.string :text
```

Comment:
These fields, don't you validate them?

Reply:
they are generated migration, validations are only done on the models

Reply:
fixed